fido_app/__init__.py

```diff
@@ -34,11 +39,53 @@
 )
 dbcursor = dbconnection.cursor()
 
-# import declared routes
-from . import routes
 
-#### DEBUGGING ####
+#from project import routes
+
+
+''' USER CLASS (for use with Flask-Login) '''
```

Comment:
I think some of this stuff, such as the `User` class and the `verify_*` functions should be placed in files other than `__init__.py` because this one is getting a bit bloated. Maybe throw the `verify_*` function into a `utils.py` file (there might already be one) and the `User` class into its own file? Just some ideas.

Reply:
Done as suggested (f86e39fb42012226bd93866c6f25c8766bc1a8f0)--now `validate_email()`, `validate_display_name()`, and `log()` are in `utils.py`, and `User` is defined in the new `user.py`.